### DISCUSSION TITLE
warning: Release Image Architecture not detected. Release Image Architecture is unknown

### DISCUSSION THREAD

**Discussion by kai-uwe-rommel** [okd-project/okd#2023] [Pre-Release Testing]
I noticed with the current nightly 4.18 build the following warning message
`level=warning msg=Release Image Architecture not detected. Release Image Architecture is unknown`
during "openshift-install generate manifests". This is for vSphere UPI on amd64, of course.
The installation did proceed normally and ended successfully. So it may not mean much, but ...

**Comment by Prashanth684** [comment]
My guess is that the `oc` binary you are using to extract the tools is pre 4.17. 4.17+ openshift clients should have the functionality to extract the release image.

**Reply by kai-uwe-rommel** [reply]
The message was issued by the OpenShift installer, as I wrote. And was issued when running "openshift-install generate manifests" on an amd64 Linux system ... so I don't think that oc had to do with this.

**Comment by Prashanth684** [comment]
the installer is extracted from the release using oc. The oc version needs to be updated to be 4.17+ to reflect the release arch. This is the functionality: https://github.com/openshift/oc/pull/1792 that was added into oc to do that

**Comment by kai-uwe-rommel** [comment]
Sounds strange, but I will give it a try. Will the downloaded installer really differ depending on the oc version with which I download it???

**Comment by Prashanth684** [comment]
> Will the downloaded installer really differ depending on the oc version with which I download it

The metadata that is written into the installer is done during extraction time, yes. so details like release payload associated with the installer and architecture.

**Comment by kai-uwe-rommel** [comment]
Indeed, by using newest OC to download the installer, I no longer saw the warning message.
